xml/System.Text.Json/JsonSerializer.xml

```diff
@@ -517,6 +517,8 @@ For more information, see [How to serialize and deserialize JSON](~/docs/standar
 
           ]]></format>
         </remarks>
+        <exception cref="T:System.ArgumentNullException">
```

Comment:
Should this be on one line? Does it make a difference one way or another?

Asking because it was brought up earlier: https://github.com/dotnet/corefx/pull/42026#discussion_r338085537

Reply:
One line when it starts with text, like this when it starts with a tag. Just to avoid having the CI touching this file with no need. But it doesn't matter really.

Reply:
So, we should leave it as is then (so that CI doesn't have to modify this file again). Sounds good.

> One line when it starts with text,

Can you give me an example of that?

Reply:
Example of a line starting with just text:

```xml
<exception cref="ArgumentException">I started with text.</exception>
```

In contrast with a line starting with a tag:
<exception cref="ArgumentException"><paramref name="lookAtMe" /> I started with a param tag.</exception>
```xml

```